docs/hypernode-platform/mysql/how-to-use-mysql-on-hypernode.md

```diff
@@ -196,6 +196,31 @@ TABLE="core_url_rewrite"
 mysql "$DATABASE" -e "TRUNCATE TABLE $TABLE"
 ```
 
+## Changing the Authentication Plugin for a MySQL User
+
+Some MySQL features and upgrades require that users use a supported authentication plugin.
+For example, newer MySQL versions no longer support `mysql_native_password` by default.
+
+To change the authentication plugin for an existing user to the recommended `caching_sha2_password` plugin, first log in to MySQL:
+
+```bash
+mysql
+```
+
+Then run the following command, replacing `<username>`, `<host>` and `<password>` with your own values:
+
+```mysql
+ALTER USER '<username>'@'<host>' IDENTIFIED WITH caching_sha2_password BY '<password>';
+```
+
+For example, to update a custom user for all hosts:
+
+```mysql
+ALTER USER 'someuser'@'%' IDENTIFIED WITH caching_sha2_password BY 'new_secure_password';
+```
+
+After changing the plugin, update your `.my.cnf` (or any other configuration files) to reflect the new password.
```

Comment:
If this isn't required for the app user, maybe good to remove this as people will start using the non-app user.